packages/wonder-blocks-grid/components/grid.js

```diff
@@ -66,7 +66,7 @@ export default class Grid extends React.Component<
     },
 > {
     static WATCHERS: {
-        [size: GridSize]: any,
+        [query: string]: any,
```

Comment:
Why can't we use `GridSize` here?

Reply:
I changed it! It's no longer keyed on the size, it's keyed on the media query string (which is more accurate).